internal/app/collector/monitors/update_global_index_monitor_test.go

```diff
@@ -122,8 +122,7 @@ func (suite *UpdateGlobalIndexMonitorTestSuite) testThresholdTxRequest(networkGe
 }
 
 func (suite *UpdateGlobalIndexMonitorTestSuite) TestThresholdTxRequest() {
-	//suite.testThresholdTxRequest(config.NetworkGenerationColumbus5)
-	suite.testThresholdTxRequest(config.NetworkGenerationColumbus4)
+	// suite.testThresholdTxRequest(config.NetworkGenerationColumbus5)
```

Comment:
why was it commented out?

Reply:
do not see particular reason, may be just forgot to uncomment it. The test seems to be working